### PR TITLE
fix(html5/akka/graphql-server): fix presentation-uploader toast and modal

### LINKED ISSUE #22177: [3.0 rc2] Presentations lost
**Describe the bug**
During today's community test we discovered that uploaded presentations have been lost. No presentations were visible in the plus menu of the presenter anymore. They weren't visible in the upload presentations dialog either. The presentation still existed in the whiteboard, but I do not remember if we could use it, switch slides etc. I have seen this in 3.0 before, I think it was 3.0 rc1, but I am not sure.

As I cannot reproduce this, I'll note what we did from memory:

1. Meeting has one moderator, all other users were guests (not viewers). I was one of the guests.
2. Moderator uploaded one presentation
3. Moderator made me presenter
4. Presenter changed a few times back and forth, I became presenter again.
5. I wanted to upload another presentation → no other presentations visible in neither menu nor presentation upload dialog
6. I uploaded a presentation → did work. Everything continued to work, with just this one uploaded presentation.

**BBB version:**

BBB 3.0 rc2

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Firefox
 - Version 128 ESR

### PR DESCRIPTION
**Pull request by GuiLeme**
### What does this PR do?

It fixes some problems in the presentation-uploader modal and toast:
- Changing the presenter would cause the new presenter to not see anything in the presentation-uploader modal (fixed);
- First upload after the `default.pdf` would cause the toast to show the `default.pdf` once again as successfully uploaded along the newly uploaded slides (fixed);
- A similar problem to the above happened with the exporting presentation to chat (fixed);
- (Not closely related but has some similarities) Converting shared-notes to PDF would break the whiteboard (fixed); 

### Closes Issue(s)

Closes #22177, #22160
The other problems don't have open issue for that, as far as I know.

### How to test

Generally speaking, I'd recommend testing those scenarios mentioned in the issues, as well as the problems I mentioned in the beginning of this PR (last 2 items). Moreover, I'd also recommend testing some edge cases you might think of!

### CODE REVIEW

**Comment by gustavotrott** [comment]
I try to close the notification of invalid presentation. But if I reload the browser it comes back every time!

[Screencast from 2025-02-07 09-59-16.webm](https://github.com/user-attachments/assets/7482123f-3ae6-4444-b500-cd4b03f7ed9e)

**Comment by gustavotrott** [comment]
It keeps showing "0 uploads complete" notification sometimes.

[Screencast from 2025-02-07 10-02-53.webm](https://github.com/user-attachments/assets/1eac40b6-f09d-4fac-baa0-32171a3b8494)

**Comment by gustavotrott** [comment]
If the user is uploading a presentation and lose the presenter status during the conversion. The toast stay in the screen forever without changing the progress.

[Screencast from 2025-02-07 10-09-22.webm](https://github.com/user-attachments/assets/ff2d37ab-5905-4e95-8081-844a583b5e52)

File: bbb-graphql-server/bbb_schema.sql [hunk 1]
```diff
@@ -1298,6 +1298,7 @@ CREATE TABLE "pres_presentation" (
 	"downloadable" boolean,
 	"downloadFileExtension" varchar(25),
 	"downloadFileUri" varchar(500),
+    "renderedInToast" boolean,
```
Comment (gustavotrott) [inline bbb-graphql-server/bbb_schema.sql:1301]: The name `renderedInToast` feels overly technical and doesn't intuitively communicate its purpose. From what I understand, this flag is meant to indicate whether a notification about this upload being complete has been shown. A clearer and more descriptive name could help make the code easier to read and maintain. For instance, I would suggest naming it `uploadCompletionNotified` or something similarly intuitive.

**Review by gustavotrott — APPROVED** [review]
Merging as is because this PR will resolve a test that is often failing.
And @GuiLeme will send a subsequent PR addressing the comments in this PR.

Good work so far!
